### PR TITLE
send email verification api

### PR DESCRIPTION
**Pull request by TdevM**

### CODE REVIEW

**Comment by championswimmer** [comment]
This pull request **introduces 1 alert** when merging e2afd445b92048ad9de0b1c55b117efdbb5498a8 into 4be0b0359776dfe2085c59d79c6fa29c42109096 - [view on LGTM.com](https://lgtm.com/projects/g/coding-blocks/oneauth/rev/pr-8763ad5dc952e913bfe77912951a559438cc27f9)

**new alerts:**

* 1 for Missing rate limiting

---

**Warning** - Automated code review for coding-blocks/oneauth will be disabled on October 1, 2019. You can avoid this by <a href="https://github.com/apps/lgtm-com/installations/new">installing the LGTM.com GitHub App</a>. Read about the <a href="https://lgtm.com/blog/announcing_github_apps_migration">benefits of migrating to GitHub Apps</a> in the blog.

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: src/routers/api/users.js [hunk 1]
```diff
@@ -300,5 +300,52 @@ router.post('/add',
...
+    makeGaEvent('POST', 'clientAPI', 'sendVerifyemail'),
+    passport.authenticate('bearer', {session: false}),
+    async (req, res, next) => {
+
+        try {
+            if (!req.body.email || req.body.email.trim() === '') {
+                req.flash('error', 'Email cannot be empty')
+                return res.status(400).json({error:' Email cannot be empty'})
+            }
+            let user = await findUserByParams({
```
Comment (witty123) [inline src/routers/api/users.js]: Could've used this user object for both the cases by using email in params instead of verifiedemail

Comment (witty123) [reply src/routers/api/users.js]: Merging for now due to immediate requirement

**Comment by championswimmer** [comment]
@witty123 	deployed ?

**Comment by witty123** [comment]
Yes.

**Pull request merged**